src/components/adherence/participants/AdherenceParticipants.tsx

```diff
@@ -52,19 +52,20 @@ const AdherenceParticipants: FunctionComponent<AdherenceParticipantsProps> =
 
     const [adherenceParams, setAdherenceParams] =
       React.useState<WeeklyAdherenceFilter>({})
-    const [currentPage, setCurrentPage] = React.useState(1)
+    const [currentPage, setCurrentPage] = React.useState(0)
+
     const [pageSize, setPageSize] = React.useState(5)
     const [sessions, setSessions] = React.useState<SessionDisplayInfo[]>([])
 
     const {
       data: adherenceWeeklyReport,
       status: adhStatus,
       error: adhError,
-    } = useAdherenceForWeek(studyId, currentPage, pageSize, adherenceParams)
+    } = useAdherenceForWeek(studyId, currentPage + 1, pageSize, adherenceParams)
```

Comment:
can we update adherence hook to also use 0-indexing? We will also need to update the service in that case